toolchain/check/testdata/class/generic/adapt.carbon

```diff
@@ -0,0 +1,912 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/class/generic/adapt.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/class/generic/adapt.carbon
+
+// --- adapt_specific_type.carbon
```

Comment:
Shouldn't this have a `fail_` prefix? Alternatively, should `Access` return `i32`, like `ImportedAccess`?

Reply:
Oops, yes, it should return `i32`. Thanks, fixed.